101-data-share-account/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create a data share account",
+  "description": "This template creates a data share account",
+  "summary": "Create a Data Share account",
+  "githubUsername": "dadihe",
+  "dateUpdated": "2020-06-29",
+  "environments":[
```

Comment:
this isn't correct, see: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#environments

Reply:
updated with default value.

Reply:
It fails build bc' it is using westus to deploy where we currently don't support. Should I update these issue somewhere? also we have 6 other folders/resource to be updated, can I include them into this PR all should I separate them one each? Thanks